website/integrations/services/pgadmin/index.md

```diff
@@ -13,89 +13,82 @@ sidebar_label: pgAdmin
 >
 > -- https://www.pgadmin.org/
 
-:::note
-This is based on authentik 2022.3.3 and pgAdmin4 6.19
-:::
-
```

Comment:
Assuming you updated the docs to reflect a more recent version of pgAdmin? If so, we should perhaps state that here.

Reply:
Good question... yeah true most other integration guides don't have it. But, it is useful data, so let's not leave it out. Anythng useful to our dear readers, let's include. Indeed, maybe one day we could get it added to the others. ;-)